graveler/ref/manager_test.go

```diff
@@ -610,6 +611,37 @@ func (f *fakeAddressProvider) ContentAddress(_ ident.Identifiable) string {
 	return res
 }
 
+func TestConsistentCommitIdentity(t *testing.T) {
+	hex := ident.NewHexAddressProvider()
+
+	commit := graveler.Commit{
+		Committer:    "some-committer",
+		Message:      "I just committed",
+		MetaRangeID:  "123456789987654321",
+		CreationDate: time.Date(2021, time.January, 24, 15, 10, 11, 0, time.UTC),
+		Parents: graveler.CommitParents{
+			graveler.CommitID("132456987153687sdfsdf"),
+			graveler.CommitID("1324569csfvdkjhcsdkjc"),
+		},
+		Metadata: map[string]string{
+			"sdkafjnb":       "1234",
+			"sdkjvcnbkjndsc": "asnjkdl",
+		},
+	}
+
+	// Should NOT be changed (unless you really know what you're doing):
+	// If this is failing and you're tempted to change this value,
+	// then you probably introducing a braking change to commit's identity.
+	// All previous references to commits (if not migrated) will be potently lost.
+	const expected = "2a7c0499ffc766c1e47aff0dd0535c8e74a9c89bfc6035339e4700737346f4e3"
+
+	const iterations = 10000
```

Comment:
Best add a comment, otherwise nobody will ever believe you why you think a digest computation can change :-)